src/selectors/getAllProjectKeys.js

```diff
@@ -1,10 +1,26 @@
 import {createSelector} from 'reselect';
+import isNull from 'lodash-es/isNull';
 
 export default createSelector(
   state => state.get('projects'),
   projects => projects.
-    filter(project => project.updatedAt).
-    sortBy(project => -project.updatedAt).
+    sort((...args) => {
+      const [firstProject, secondProject] = args;
+      const firstProjectUpdatedAt = firstProject.updatedAt;
+      const secondProjectUpdatedAt = secondProject.updatedAt;
+      const isFirstProjectNull = isNull(firstProjectUpdatedAt);
+      const isSecondProjectNull = isNull(secondProjectUpdatedAt);
```

Comment:
I would probably call these `isFirstProjectPristine` etc. to avoid confusion (the project itself is not null).